README.md

```diff
@@ -93,7 +118,8 @@ cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=.. -DLLVM_TARGETS_TO_BUILD= -DL
 make -j4 install
 ```
 
```

Comment:
Can we add a link to skip past the macOS section here? Something like "Your build environment is set up, click here to skip to 'Building EOS'"

Reply:
Yeah, added